corehq/apps/cloudcare/static/cloudcare/js/form_entry/entrycontrols_full.js

```diff
@@ -373,6 +373,9 @@ function ComboboxEntry(question, options) {
                 matcher: function() {
                     return $input.val();
                 },
+                sorter: function(query, data) {
+                    return data;
+                },
```

Comment:
Yeah, surely a sorter isn't supposed eliminate anything; it just moves the items the user is most likely to want up to the top of the already-filtered items, right?